client/src/components/molecules/Table.js

```diff
@@ -24,10 +37,21 @@ function createBasicData(jobId, title, dateCreated, matches) {
   return { jobId, title, dateCreated, matches };
 }
 
-function createData(isStarred, jobId, title, dateCreated, matches) {
+function createStarredData(isStarred, jobId, title, dateCreated, matches) {
   return { isStarred, jobId, title, dateCreated, matches };
```

Comment:
do we no longer want to return isStarred?

Reply:
This table was initially designed for the intention of having bookmarked (starred) job postings. Considering we aren't using this. I'm just gonna remove it